### ISSUE TITLE
Can't change icon if installed from package

### ISSUE THREAD

**Issue by hotice** [SergKolo/launcher-list-indicator#5]
If installed from a deb package (with sudo, to a system dir), the icon can't be changed since Launcher List Indicator tries perform file operations in the folder where it's installed (like renaming the original icon, etc.), and it fails. An alternate solution to fixing this would be to select from the colored and mononchrome icon directly from its settings.

I've created a package for launcher-list-indicator in the main WebUpd8 PPA: https://launchpad.net/~nilarimogard/+archive/ubuntu/webupd8/+packages?field.name_filter=launcher-list-indicator&field.status_filter=published&field.series_filter= and fixing this would make this feature work when installing the package from the PPA (and it would also be fixed if later it will be included in the Ubuntu repositories, etc.).

**Comment by SergKolo** [comment]
@hotice  where does the deb package place the icon?  I believe  this can be easily fixed by checking if icon exists in same directory,  else set custom path

**Comment by hotice** [comment]
The icon is placed in the same directory as the python script (in /opt/launcher-list-indicator). 

Ideally (and how most AppIndicators behave), the icon would be placed in /usr/share/icons/... and allow icon themes to change it, but for this, launcher-list-indicator would need to access its icon from the current icon theme folder.

**Comment by SergKolo** [comment]
@hotice 　Ah, I see what is happening there. Each user has `~/.local/share/icons`. I could change the code to look for new icon in that directory. Indicator , which is running as user, should be able to read and change items in that folder. Would that be a sufficient solution ?

**Comment by hotice** [comment]
I'm not sure what your exact solution is. A package installed with sudo can't access (well, it can with some tricks, but it shouldn't and it's a pretty bad thing to do) a user's home folder, so the package can't install the icons to ~/.local/share/icons/. Since the app runs as user, launcher-list-indicator can copy the icon to ~/.local/share/icons and use it from there, so yes, that would be a solution. 

Basically, this would work: by default, use the icon from the launcher-list-indicator installation folder (for example, for my package, that would be: /opt/launcher-list-indicator/); if the user selects another icon, you can copy it to ~/.local/share/icons and read it from there.

**Comment by SergKolo** [comment]
@hotice  yes, your last paragraph describes exactly what I was meant to say. I'll work on it, push update, probably today. I'll let you know once it's done

**Comment by SergKolo** [comment]
@hotice  I've implemented the copying to `~/.local/share/icons/launcher-list-indicator` . Let me know what you think

**Comment by hotice** [comment]
It works, thank you for the quick fix!

**Comment by SergKolo** [comment]
@hotice  always welcome. I'm not the best programmer but trying to deliver the best results i can :)
